test/test_resample.py

```diff
@@ -71,7 +71,7 @@ def test_resample_errors(self):
     def test_lfilter(self):
```

Comment:
For me I think the proof of this, is here in this test function. Are you satisfied that this tests all possible inputs? I think this would test the recursive code path, which would also end up in the ` < 2**18` code path as well. Not sure if we need to test the "fall back to scipy" path, as scipy is the reference, but perhaps it wouldn't hurt to do so?